packages/expo-modules-autolinking/src/platforms/android.ts

```diff
@@ -135,9 +161,24 @@ async function findAndroidPackagesAsync(modules: ModuleDescriptor[]): Promise<st
 }
 
 /**
- * Converts the package name to Android's project name.
- * Example: `@unimodules/core` → `unimodules-core`
+ * Converts the package name and gradle file path to Android's project name.
+ *   `$` to indicate subprojects
+ *   `/` path will transform as `-`
+ *
+ * Example: `@unimodules/core` + `android/build.gradle` → `unimodules-core`
+ *
+ * Example: multiple projects
+ *   - `expo-test` + `android/build.gradle` → `react-native-third-party`
+ *   - `expo-test` + `subproject/build.gradle` → `react-native-third-party$subproject`
+ *
+ * Example: third party expo adapter module
+ *   - `react-native-third-party` + `expo/android/build.gradle` → `react-native-third-party$expo-android`
```

Comment:
I guess we no longer need this example — it's already well-described by the examples for multiple projects.
```suggestion
```